src/main/java/hw2/Task1.java

```diff
@@ -0,0 +1,22 @@
+package hw2;
+
+/*
+1.	Write a Java program to concatenate a given string to the end of another string. And remove all `P` or `p` characters.
+Test Data:
+String 1: PHP Exercises and
+String 2: Python Exercises
+Expected output:
+The concatenated string with replacements: H Exercises and ython Exercises
+
+ */
+public class Task1 {
+    public String conctenate() {
+
+
+        String a = "PHP Exercises and ";
```

Comment:
Please convert "a" and "b" variables to be a part of method's arguments according to the task. these string were just an example of expected output for the given strings

Reply:
test